.github/workflows/e2e-functional-template.yml

```diff
@@ -186,18 +187,41 @@ jobs:
           npx electron-rebuild --platform=win32 -f -t prod,optional,dev -w robotjs --module-dir ../
           npm install mochawesome-report-generator
 
-      - name: e2e/run-playright-tests-${{ runner.os }}
+      - name: e2e/run-playright-tests-linux
+        if: runner.os == 'Linux'
+        run: |
+          export DISPLAY=:99
+          Xvfb $DISPLAY -screen 0 1280x960x24 > /dev/null 2>&1 &
+          npm run build-test
+          cd e2e
+          xvfb-run -a npm run run:e2e || true
+          npm run send-report
+        env:
+          SERVER_VERSION: ${{ inputs.MM_SERVER_VERSION }}
+          DESKTOP_VERSION: ${{ inputs.DESKTOP_VERSION }}
+
+      - name: e2e/run-playright-tests-macos
+        if: runner.os == 'macOS'
+        run: |
+          sudo spctl --master-disable
+          xattr -dr com.apple.quarantine ./node_modules/electron
```

Comment:
disables the macOS security feature that restricts running apps from untrusted sources.